karma-ng.conf.js

```diff
@@ -20,7 +20,8 @@ function makeConfig(packageName, argv) {
   /* eslint complexity: [0] */
   const launchers = makeBrowsers(packageName, argv);
   const integrationTestPath = path.join('packages', 'node_modules', packageName, 'test', 'integration', 'spec', '**', '*.js');
-  const unitTestPath = path.join('packages', 'node_modules', packageName, 'test', 'unit', 'spec', '**', '*.js');
+  const unitTestPath = path.join('packages', 'node_modules', packageName, 'src', '**', '*-spec.js');
+  const legacyUnitTestPath = path.join('packages', 'node_modules', packageName, 'test', 'unit', 'spec', '**', '*.js');
```

Comment:
@adamweeks is this change still necessary/helpful? 
@taymoork2 will this change still jive with our tests?

Reply:
@mattnorris no clue to be honest. I think we had originally wanted to change the unit test files to `-spec.js`, but I don't see the need for it now. Probably can close this and maybe take some of the doc changes into a new PR